ocaml-variants.opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "ocaml-variants"
+version: "4.08.0+dev"
+synopsis: "OCaml development version"
+depends: [
+  "ocaml" {= "4.08.0" & post}
```

Comment:
IIRC, we can put `= version` here, and it'll do the right thing directly.

Reply:
I have just tried it with `
depends: [
  "ocaml" {= "%{version}%" & post} ...`
It fails when I try to create a local repo from a global system switch that has 4.07.1. 
```
The following dependencies couldn't be met:
  - ocaml-variants -> ocaml >= 4.08.0+trunk
      no matching version
```
Is this the behavior you expect? With the explicit dependency on 4.08.0 & post, opam finds ocaml.opam (currently in the same directory) and installs ocaml-variants successfully.

Reply:
Ah, wait, the version is `4.08.0+trunk`, but we want `4.08.0`. Nevermind, disregard that remark. :)